backend/app/verification.py

```diff
@@ -884,6 +912,78 @@ def unverify(
         worker.republish_app.send(app_id)
 
 
+@router.post("/{app_id}/enroll", status_code=204, tags=["verification"])
+def enroll(
+    login=Depends(logged_in),
+    app_id: str = Path(
+        min_length=6,
+        max_length=255,
+        pattern=r"^[A-Za-z_][\w\-\.]+$",
+        examples=["org.gnome.Glade"],
+    ),
+):
+    is_direct_upload = models.DirectUploadApp.by_app_id(sqldb, app_id) is not None
```

Comment:
I'm failing a bit to figuring this out.

As far as I know, a direct upload app can't be created with the same `app_id` as an already existing app. So this and having a github repo to archive is impossible?

Reply:
But endpoints are public, ale people usually do dumb things when they can, so it's better safe than sorry here.

Reply:
I guess what I meant was an api check - and `_create_direct_upload_app` would not fail due to an already existing github app, as we don't enforce that in the database - still slightly worried about a directupload with the same appid as a githubrepo, but I guess that's where the archived flag comes into play.